.clang-format

```diff
@@ -0,0 +1,16 @@
+---
+BasedOnStyle: WebKit
+AllowShortBlocksOnASingleLine: false
+AllowShortFunctionsOnASingleLine: false
```

Comment:
What version of clang-format are you using here?  See below where mine is no longer a boolean ("None").

Reply:
I am using clang-format 6.0